### PR TITLE
Adding support to supply custom connection manager

### PR DESCRIPTION
**Pull request by SaikiranDaripelli**
Pull request for #802 on master branch

### CODE REVIEW

File: redisson/src/main/java/org/redisson/config/Config.java [hunk 1]
```diff
@@ -296,6 +302,15 @@ ReplicatedServersConfig getReplicatedServersConfig() {
     void setReplicatedServersConfig(ReplicatedServersConfig replicatedServersConfig) {
         this.replicatedServersConfig = replicatedServersConfig;
     }
+    
+    ConnectionManager getConnectionManager() {
+        return connectionManager;
+    }
+
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/config/Config.java]: @SaikiranDaripelli  Please add comment for this method

Comment (SaikiranDaripelli) [inline redisson/src/main/java/org/redisson/config/Config.java]: @mrniko updated the pull request with comments.

**Comment by mrniko** [comment]
Thanks!

**Comment by SaikiranDaripelli** [comment]
@mrniko any tentative timeline for release?

**Comment by mrniko** [comment]
During next week

**Comment by SaikiranDaripelli** [comment]
@mrniko are snapshot jars available in any maven repo?

**Comment by mrniko** [comment]
no, only releases

**Pull request merged**